src/server/session.ts

```diff
@@ -553,10 +564,7 @@ module ts.server {
                     compilerService.host.editScript(file, start, end, insertString);
                     this.changeSeq++;
                 }
-                // update project structure on idle commented out
-                // until we can have the host return only the root files
-                // from getScriptFileNames()
-                //this.updateProjectStructure(this.changeSeq, (n) => n == this.changeSeq);
+                this.updateProjectStructure(this.changeSeq, (n) => n == this.changeSeq);

```

Comment:
We usually don't include parens around the parameter list for single-argument arrow functions. That occurs here, line 201, and 188.